tests/test_benchmark.py

```diff
@@ -1022,7 +1040,13 @@ def test_zzz_lzma(self):
     src = read_file(test_file('benchmark', 'test_lzma_benchmark.c'))
 
     def lib_builder(name, native, env_init):
-      return self.get_library(os.path.join('third_party', 'lzma'), [os.path.join('lzma.a')], configure=None, native=native, cache_name_extra=name, env_init=env_init)
+      return self.get_library(os.path.join('third_party', 'lzma'),
```

Comment:
I think we might want to do something like `make = ['make'] if native else [EMMAKE, 'make']` so continue to support the native build.   We could perhaps find a way to make that the default.

Reply:
Actually I think I have better fix..